services/account_type/legacy.yaml

```diff
@@ -79,12 +79,7 @@ Resources:
     Type: AWS::IAM::Role
     Condition: CreateResources
     Properties:
-      Tags:
-        - Key: cloudzero-stack
-          Value: !Ref AWS::StackName
-        - Key: cloudzero-reactor-account-id
-          Value: !Ref ReactorAccountId
```

Comment:
Are tags assigned by the stack itself?

Reply:
The stacks _do_ have tags. It does not look like the `Role`s have tags =(. They are under the `/cloudzero/` `Path` at least.

Reply:
It's very weird, b/c this `Tags:` CFN property _works_ in `us-east-1` but not in _all_ other regions I've tried.